pgtap/depthFirstSearch/issue1348.sql

```diff
@@ -20,7 +20,7 @@ PREPARE q0 AS
 SELECT id, source, target, cost, reverse_cost FROM sample_table
 ORDER BY id;
 
--- Directed Graph with Root vid 3
+
 PREPARE q1 AS
```

Comment:
In issue 1348: 

> Query 1 (Directed)

As this is about that issue, somehow incorporate the titles of the queries that are mentioned in the issue

maybe the message on the tap test.